### ISSUE TITLE
Increased esterified to free carnitine ratio

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6424]
**Preferred term label:** Increased esterified to free carnitine ratio

**Synonyms**

**Definition (free text, please give PubMed ID)**  A relatively high amount of esterified carnitine when compared to free carnitine in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating carnitine concentration HP:0010967

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
